### PR TITLE
[Multilingual Bot] Changed documentation

### PR DESCRIPTION
**Pull request by matiasroldan6**
## Description
Changed errors and unspecificities in multilingual bots documentation (JS and .NET).
This addresses issue number 2001 in BotBuilder-Samples.

## Proposed Changes
Documentation has been changed to reflect the current functionality of the multilingual sample bot both in [JS](https://github.com/microsoft/BotBuilder-Samples/tree/master/samples/javascript_nodejs/17.multilingual-bot) and [.NET](https://github.com/microsoft/BotBuilder-Samples/tree/master/samples/csharp_dotnetcore/17.multilingual-bot). Also, a code snippet has been fixed.

### CODE REVIEW

**Comment by Aliandi** [comment]
Change to:

> Documentation has been changed to reflect the current functionality of the multilingual sample bot (link to the bot), both in JS and .NET. Also, a code snippet has been fixed.

As:
- Limited sounds too negative
- You should LINK to the bot you are fixing
- I think prenthesis are not necessary
- `a wrong code snipet` is not correct

File: samples/csharp_dotnetcore/17.multilingual-bot/README.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 Bot Framework v4 multilingual bot sample
 
+This sample will present you with a set of cards to select your choice of language. You can either change language by invoking the option cards, or by entering the language code (en, es). The sample will then acknowledge your selection.
```
Comment (Aliandi) [inline samples/csharp_dotnetcore/17.multilingual-bot/README.md]:
```suggestion
This sample will present you with a set of cards to select your choice of language. You can either change language by invoking the option cards, or by entering the language's code (e.g, _en_, _es_).  Then, the bot will then acknowledge your selection.
```

Comment (Aliandi) [reply samples/csharp_dotnetcore/17.multilingual-bot/README.md]: (For both readme.md)

**Review by Aliandi — CHANGES_REQUESTED** [review]

File: samples/csharp_dotnetcore/17.multilingual-bot/README.md (hunk 1 above)
Comment (matiasroldan6) [reply samples/csharp_dotnetcore/17.multilingual-bot/README.md]: I have to remove 1 `then` from that last sentence

**Review by ceciliaavila — APPROVED** [review]

**Pull request closed without merging**